seed/seed.go

```diff
@@ -157,6 +157,9 @@ type Seed interface {
 	// Iter provides a way to iterately perform a function on
 	// each of the snaps for which LoadMeta loaded their metadata.
 	Iter(f func(sn *Snap) error) error
+
+	// Auto import assertion from root of seed if grade is dangerous
```

Comment:
this needs a . at the end and also needs  to start with the name of the method to follow golang conventions. Also I wonder if it should return error at all given that it actually doesn't except for 16? This should probably explain that it does a best-effort and doesn't actually error but also this a UC20+ seed only feature, what we did in the past would be not to have this on Seed at all but have a separate dedicated interface AutoImportAssertionsLoaderSeed  (see 65e1457ac7781 for an example of this kind of approach).

Reply:
@pedronis  I have now moved the functionality into dedicated interface as suggested and also removed error return value, as indeed it was only erroring on UC16.